traffic_portal/app/src/modules/public/sso/SsoController.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+var SsoController = function($scope, $state, $interval, urlUtils, authService, propertiesModel) {
+
+	var init = function () {
+        const queryParams = urlUtils.getUrlQueryParams(window.location.href);
+        const tokenParameter = propertiesModel.properties.oAuth.oAuthTokenQueryParam;
+        const token = queryParams[tokenParameter];
+
+        authService.oauthLogin(token)
+            .then(
+                function() {}
```

Comment:
If you're not doing anything, do you need a `.then`? Maybe just log a success message or something?

Reply:
done